examples/table/main.go

```diff
@@ -22,6 +22,8 @@ func (m model) Init() tea.Cmd { return nil }
 func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	var cmd tea.Cmd
 	switch msg := msg.(type) {
+	case tea.WindowSizeMsg:
+		m.table.SetHeight(msg.Height - 1)
```

Comment:
The `1` here is a magic number: why is it needed?

Reply:
hahaha legit no reason at all, just updated it to remove that and it works great